app/views/layouts/base.html.erb

```diff
@@ -19,7 +19,8 @@
   <%= preload_link_tag font_url('public-sans/PublicSans-Bold.woff2') %>
   <%= preload_link_tag font_url('public-sans/PublicSans-Regular.woff2') %>
   <%= render_stylesheet_once_tags %>
-  <%= stylesheet_link_tag 'application', media: 'all' %>
+  <%= stylesheet_link_tag 'application' %>
```

Comment:
does it make a difference to keep the `media: 'all'` or is that basically the default value?

Reply:
I think it's redundant. I couldn't find anything which explicitly states that it's the default, but this documents the distinction between `print` and `screen`, and `all` as matching the union. I assume we meant it as applying the same as if it weren't applied at all.

https://drafts.csswg.org/mediaqueries/#media-types